app/views/two_factor_authentications/index.html.slim

```diff
@@ -4,47 +4,56 @@
   .error-group--wrapper.js-authenticate-u2f-error
 
     .js-u2f-error-bad-request
-      .single-panel--content.single-panel--content--warning.icon-and-text--wrapper
+      .single-panel--content.single-panel--content--alert.icon-and-text--wrapper
         .icon-and-text--icon
-          = render 'icon_circled_x'
+          = render "icon_circled_x"
         .icon-and-text--text
           = t ".u2f-error.bad-request"
 
     .js-u2f-error-configuration-unsupported
-      .single-panel--content.single-panel--content--warning.icon-and-text--wrapper
+      .single-panel--content.single-panel--content--alert.icon-and-text--wrapper
```

Comment:
These were incorrectly in warning styles before. Should be alert/error/red style.